<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addHotelDialog</class>
 <widget class="QDialog" name="addHotelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nest</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>  欢迎注册新酒店:D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>酒店名字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>酒店地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Location">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>酒店星级：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_star">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>69</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25  12pt &quot;Calibri Light&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>   2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>   3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>   4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>   5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>   6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>   7</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;
</string>
   </property>
   <property name="text">
    <string>酒店描述：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Feature">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;
</string>
   </property>
   <property name="text">
    <string>营业许可号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Id">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_regi">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
